web/js/components/sidebar/event.js

```diff
@@ -51,6 +51,31 @@ function Event (props) {
     }
   }
 
+  /**
+   *
+   * @param {Object} geometry | Geometry object containing magnitude data
+   * @returns Magnitude data output
+   */
+  function magnitudeOutput({ magnitudeUnit = null, magnitudeValue = null }) {
+    const unit = magnitudeUnit || null;
+    let value = magnitudeValue ? magnitudeValue.toLocaleString() : null;
+    if (!unit && !value) return;
+
+    const formattedunit = unit === 'kts' ? ' kts' : ' NM';
+    value = value.toLocaleString();
+    return (
+      <p className="magnitude">
+
+        {formattedunit === ' NM' ? 'Surface Area: ' : 'Wind Speed: '}
+        {value}
+        {formattedunit}
+        {formattedunit === ' NM' && (
+          <sup>2</sup>
+        )}
+      </p>
+    );
+  }
+
```

Comment:
This works well but I think it can be simplified a bit.  You are setting param defaults to `null` but then also checking for whether or not they have a defined value and setting to null again if they don't which is redundant.  I think you can avoid a lot of this by just checking if either parameters are falsy as the first line and bail out if either are:

```
 function magnitudeOutput({ magnitudeUnit, magnitudeValue }) {
    if (!magnitudeUnit || !magnitudeValue) return;
    const formattedunit = magnitudeUnit === 'kts' ? ' kts' : ' NM';
    return (
      <p className="magnitude">
        {formattedunit === ' NM' ? 'Surface Area: ' : 'Wind Speed: '}
        {magnitudeValue.toLocaleString()}
        {formattedunit}
        {formattedunit === ' NM' && (
          <sup>2</sup>
        )}
      </p>
    );
  }
```